rotkehlchen/tests/api/test_assets.py

```diff
@@ -919,6 +919,63 @@ def test_search_assets_with_levenshtein(rotkehlchen_api_server):
     )
     assert_error_response(response, contained_in_msg='Failed to deserialize ChainID value near')
 
+    with rotkehlchen_api_server.rest_api.rotkehlchen.data.db.user_write() as cursor:
+        cursor.execute('INSERT INTO assets VALUES (?)', ('my-nft-identifier',))
+        cursor.execute(
+            'INSERT INTO nfts(identifier, name, collection_name, manual_price, is_lp, usd_price) '
+            'VALUES (?, ?, ?, ?, ?, ?)',
+            ('my-nft-identifier', 'super-duper-nft', 'Bitcoin smth', False, False, '100'),
+        )
+
+    # check that searching by nft name works
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetssearchlevenshteinresource',
+        ),
+        json={
+            'value': 'super-duper',
+            'limit': 50,
+            'search_nfts': True,
+        },
+    )
+    result = assert_proper_response_with_result(response)
+    assert result == [{
+        'identifier': 'my-nft-identifier',
+        'name': 'super-duper-nft',
+        'collection_name': 'Bitcoin smth',
+        'asset_type': 'nft',
+    }]
+
+    # Check that:
+    # 1. Searching by nft collection name works
+    # 2. Nfts are searched only with search_nfts set to True
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetssearchlevenshteinresource',
+        ),
+        json={
+            'value': 'Bitcoin',
+            'limit': 50,
+        },
+    )
+    results_without_nfts = [x['identifier'] for x in assert_proper_response_with_result(response)]
+
+    response = requests.post(
+        api_url_for(
+            rotkehlchen_api_server,
+            'assetssearchlevenshteinresource',
+        ),
+        json={
+            'value': 'Bitcoin',
+            'limit': 50,
+            'search_nfts': True,
+        },
+    )
+    results_with_nfts = [x['identifier'] for x in assert_proper_response_with_result(response)]
+    assert set(results_with_nfts) - set(results_without_nfts) == {'my-nft-identifier'}
```

Comment:
you shoud also add another part of this test to test that searching for a half-input value for many similar input provides them in the right order (following levensthein) and the order makes sense